build_bundle_image.sh

```diff
@@ -271,13 +276,31 @@ function push_docker_image {
 function set_parent_image {
 	if (echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | grep -q "q")
 	then
-		return
-	fi
+		if [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1)" -gt 2024 ]]
+		then
+			return
+		fi
 
-	if [ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION%-*}" | cut -f1,2,3 -d'.' | cut -f1 -d '-' | sed 's/\.//g' )" -le 7310 ]
+		if [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1)" -eq 2024 ]] &&
+		   [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 2 | tr -d q)" -ge 3 ]]
+		then
+			return
+		fi
+
+		sed -i 's/liferay\/jdk21:latest AS liferay-jdk21/liferay\/jdk11:latest AS liferay-jdk11/g' "${TEMP_DIR}"/Dockerfile
+		sed -i 's/FROM liferay-jdk21/FROM liferay-jdk11/g' "${TEMP_DIR}"/Dockerfile
+	elif [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1,2)" == 7.4* ]] ||
+	     [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1,2)" == 7.3* ]]
```

Comment:
@natocesarrego, since it only gets the 1,2 elements (7.3, for example), I don't think this `*` is necessary.

Reply:
It makes sense to me! Good catch! :smile: